doc/rofi-theme.5.markdown

```diff
@@ -682,22 +681,22 @@ The following widgets names are 'fixed' widgets with functionality:
  * listview
  * sidebar
 
-The following exists and automatically pack a subset of the widgets as in the above picture:
+The following exist and automatically pack a subset of the widgets as in the above picture:
```

Comment:
"The following exist and automatically pack a subset of the widgets as in the above picture:"
Unclear sentence.
Suggestion: "The following (options? widgets?) are available, and automatically pack a subset of the widgets as specified above:"